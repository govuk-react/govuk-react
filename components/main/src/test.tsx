import React from 'react';
import { mount } from 'enzyme';

import { Main } from '.';

describe('Main', () => {
  let wrapper;

  it('renders without crashing', () => {
    wrapper = mount(<Main />);
  });

  it('can render a child node', () => {
    wrapper = mount(
      <Main>
        <span>Example</span>
      </Main>
    );
    expect(wrapper.find('span')).toHaveLength(1);
  });

  it('matches wrapper snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
