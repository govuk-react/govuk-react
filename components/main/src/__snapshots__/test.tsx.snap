// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main matches wrapper snapshot 1`] = `
<Main>
  <src__OuterContainer>
    <div
      className="src__OuterContainer-sc-1lvyroj-0 gCrlSn"
    >
      <src__InnerContainer>
        <div
          className="src__InnerContainer-sc-1lvyroj-1 joqrYg"
        >
          <span>
            Example
          </span>
        </div>
      </src__InnerContainer>
    </div>
  </src__OuterContainer>
</Main>
`;
